package.json

```diff
@@ -37,6 +37,7 @@
     "electron-packager": "^6.0.0",
     "electron-winstaller": "^2.0.5",
     "gh-release": "^2.0.3",
+    "nobin-debian-installer": "^0.0.5",
```

Comment:
Btw, don't worry about the `async` dep in this package. devDependencies aren't included in the packaged application, so it matters a lot less.